autosubmit/history/database_managers/experiment_history_db_manager.py

```diff
@@ -20,439 +20,501 @@
+            IF NOT EXISTS job_data (
+            id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+            counter INTEGER NOT NULL,
+            job_name TEXT NOT NULL,
+            created TEXT NOT NULL,
+            modified TEXT NOT NULL,
+            submit INTEGER NOT NULL,
+            start INTEGER NOT NULL,
+            finish INTEGER NOT NULL,
+            status TEXT NOT NULL,
+            rowtype INTEGER NOT NULL,
+            ncpus INTEGER NOT NULL,
+            wallclock TEXT NOT NULL,
+            qos TEXT NOT NULL,
+            energy INTEGER NOT NULL,
+            date TEXT NOT NULL,
+            section TEXT NOT NULL,
+            member TEXT NOT NULL,
+            chunk INTEGER NOT NULL,
+            last INTEGER NOT NULL,
+            platform TEXT NOT NULL,
+            job_id INTEGER NOT NULL,
+            extra_data TEXT NOT NULL,
+            nnodes INTEGER NOT NULL DEFAULT 0,
+            run_id INTEGER,
+            MaxRSS REAL NOT NULL DEFAULT 0.0,
+            AveRSS REAL NOT NULL DEFAULT 0.0,
+            out TEXT NOT NULL,
+            err TEXT NOT NULL,
+            rowstatus INTEGER NOT NULL DEFAULT 0,
+            children TEXT,
+            platform_output TEXT,
+            UNIQUE(counter,job_name)
+            );
+            ''')
+        self.create_index_query = textwrap.dedent('''CREATE INDEX IF NOT EXISTS
+        ID_JOB_NAME ON job_data(job_name);''')
+
+    def _set_schema_changes(self):
+        # type : () -> None
+        """ Creates the list of schema changes"""
+        self.version_schema_changes = [
+            "ALTER TABLE job_data ADD COLUMN nnodes INTEGER NOT NULL DEFAULT 0",
+            "ALTER TABLE job_data ADD COLUMN run_id INTEGER"
+        ]
+        # Version 15
+        self.version_schema_changes.extend([
+            "ALTER TABLE job_data ADD COLUMN MaxRSS REAL NOT NULL DEFAULT 0.0",
+            "ALTER TABLE job_data ADD COLUMN AveRSS REAL NOT NULL DEFAULT 0.0",
+            "ALTER TABLE job_data ADD COLUMN out TEXT NOT NULL DEFAULT ''",
+            "ALTER TABLE job_data ADD COLUMN err TEXT NOT NULL DEFAULT ''",
+            "ALTER TABLE job_data ADD COLUMN rowstatus INTEGER NOT NULL DEFAULT 0",
+            "ALTER TABLE experiment_run ADD COLUMN suspended INTEGER NOT NULL DEFAULT 0",
+            "ALTER TABLE experiment_run ADD COLUMN metadata TEXT"
+        ])
+        # Version 16
+        self.version_schema_changes.extend([
+            "ALTER TABLE experiment_run ADD COLUMN modified TEXT"
+        ])
+        # Version 17
+        self.version_schema_changes.extend([
+            "ALTER TABLE job_data ADD COLUMN children TEXT",
+            "ALTER TABLE job_data ADD COLUMN platform_output TEXT"
+        ])
+
+    def create_historical_database(self):
+        """ Creates the historical database with the latest changes. """
+        self.execute_statement_on_dbfile(self.historicaldb_file_path,
+                                         self.create_table_header_query)
```

Comment:
After a comma, the indentation normally goes all the way back. Only keeping the same level when you have

```function(
  var,
  var,
  var
)```

otherwise it's normally

```
function (var, var,
    var, var)
```

This now adds a new warning in PyCharm

![image](https://github.com/user-attachments/assets/05b5768f-02de-4b37-9c6e-ef27ff97619e)